src/XMLimport.cpp

```diff
@@ -901,42 +914,46 @@ void XMLimport::readHostPackage()
-    pT->mAcceptServerGUI = ( attributes().value("mAcceptServerGUI") == "yes" );
-    pT->mMapperUseAntiAlias = ( attributes().value("mMapperUseAntiAlias") == "yes" );
-    pT->mFORCE_MXP_NEGOTIATION_OFF = ( attributes().value("mFORCE_MXP_NEGOTIATION_OFF") == "yes" );
-    pT->mRoomSize = attributes().value("mRoomSize").toString().toInt();
-    if (!pT->mRoomSize)
-        pT->mRoomSize=3;
-    pT->mLineSize = attributes().value("mLineSize").toString().toInt();
-    if (!pT->mLineSize)
-        pT->mLineSize=1;
-    pT->mBubbleMode = ( attributes().value("mBubbleMode") == "yes" );
-    pT->mShowRoomID = ( attributes().value("mShowRoomIDs") == "yes" );
-    pT->mShowPanel = ( attributes().value("mShowPanel") == "yes" );
-    pT->mHaveMapperScript = ( attributes().value("mHaveMapperScript") == "yes");
+    pHost->mAutoClearCommandLineAfterSend = ( attributes().value("autoClearCommandLineAfterSend") == "yes" );
+    pHost->mDisableAutoCompletion = ( attributes().value("disableAutoCompletion") == "yes" );
+    pHost->mPrintCommand = ( attributes().value("printCommand") == "yes" );
+    pHost->set_USE_IRE_DRIVER_BUGFIX( attributes().value("USE_IRE_DRIVER_BUGFIX") == "yes" );
+    pHost->mUSE_FORCE_LF_AFTER_PROMPT = ( attributes().value("mUSE_FORCE_LF_AFTER_PROMPT") == "yes" );
+    pHost->mUSE_UNIX_EOL = ( attributes().value("mUSE_UNIX_EOL") == "yes" );
+    pHost->mNoAntiAlias = ( attributes().value("mNoAntiAlias") == "yes" );
+    pHost->mIsNextLogFileInHtmlFormat = ( attributes().value("mRawStreamDump") == "yes" );
+    pHost->mAlertOnNewData = ( attributes().value("mAlertOnNewData") == "yes" );
+    pHost->mFORCE_NO_COMPRESSION = ( attributes().value("mFORCE_NO_COMPRESSION") == "yes" );
+    pHost->mFORCE_GA_OFF = ( attributes().value("mFORCE_GA_OFF") == "yes" );
+    pHost->mFORCE_SAVE_ON_EXIT = ( attributes().value("mFORCE_SAVE_ON_EXIT") == "yes" );
+    pHost->mEnableGMCP = ( attributes().value("mEnableGMCP") == "yes" );
+    pHost->mEnableMSDP = ( attributes().value("mEnableMSDP") == "yes" );
+    pHost->mMapStrongHighlight = ( attributes().value("mMapStrongHighlight") == "yes" );
+    pHost->mLogStatus = ( attributes().value("mLogStatus") == "yes" );
+    pHost->mEnableSpellCheck = ( attributes().value("mEnableSpellCheck") == "yes" );
+    pHost->mShowInfo = ( attributes().value("mShowInfo") == "yes" );
+    pHost->mAcceptServerGUI = ( attributes().value("mAcceptServerGUI") == "yes" );
+    pHost->mMapperUseAntiAlias = ( attributes().value("mMapperUseAntiAlias") == "yes" );
+    pHost->mFORCE_MXP_NEGOTIATION_OFF = ( attributes().value("mFORCE_MXP_NEGOTIATION_OFF") == "yes" );
+    pHost->mRoomSize = attributes().value("mRoomSize").toString().toFloat();
+    if( qFuzzyCompare( 1.0 + pHost->mRoomSize, 1.0 ) ) {
+        // The value is a float/double and the prior code using "== 0" is a BAD
+        // THING to do with non-integer number types!
+        pHost->mRoomSize= 0.5f; // Same value as is in Host class initalizer list
```

Comment:
It does, because the failure to read a valid value from the file here ___now___ tries to adopt the default values, which is what you have/will be tweaking.